.grit/patterns/js/iots_to_zod.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Transform `io-ts` to `zod schema`
+---
+
+Transform `io-ts` schemas to `zod` schema
+
+tags: #migration, #js, #zod, #io-ts
+
+```grit
+engine marzano(0.1)
+language js
+
+or {
+    `import * as $alias from "io-ts"` => `import z from 'zod'` where {
+        $program <: contains bubble($alias) or {
+            `$alias.$type($val)` as $exp => `z.$type($val)` where or {
+                $type <: `type` => `object`,
+                $exp <: `$alias.partial($val)`  => `z.object($val).partial()`,
```

Comment:
This is a bit confusing why you're sometimes rewriting the `type` and sometimes the whole object. Try to make it consistent.

Reply:
Make sense! I've simplified and removed redundant patterns. Thanks!